public/account.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="ru">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Spotify</title>
+        <link rel="stylesheet" href="../src/index.css"/>
+    </head>
+    <body>
+        <div class="app">
+            <header class="header">
+                <div class="header__row">
+                    <a class="header__item" href=".">
+                        <div class="header__home-image">
+                            <img src="src/common.blocks/header/home-image.png">
```

Comment:
Все содержимое из public/src можно перенести в src
и использовать пути
```suggestion
                            <img src="%PUBLIC_URL%/common.blocks/header/home-image.png">
```

Это относится и к другим местам.
Так с точки зрения архитектуры будет правильно
и запускать проект нужно будет так:
`npm run build`
`npm start`

Reply:
Все пытаюсь понять, как включить содержимое папки src, чтобы их можно было использовать после сборки и запуска приложения. Простое копирование в папку src не помогло, при обращении к картинке через
`<img src="%PUBLIC_URL%/common.blocks/header/home-image.png">` не помогает. Пытался загуглить, как включить содержимое папки в сборку, но не вышло, картинка отображается только, если она находится в public директории. Можно попросить совета, как с этим быть?

Reply:
На начальном этапе можно все картинки засунуть в директорию public